app/screens/home/search/initial/recent_searches/recent_searches.tsx

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Nothing changed in this file, except the team name is passed in as a prop instead of getting it from the database with the teamId.